src/image.rs

```diff
@@ -209,9 +224,7 @@ impl<T> ImageStore<T> {
         Ok(ImageId(self.0.insert((info, image))))
     }
 
-    ///
-    /// Reallocates the image without changing the id.
-    ///
+    // Reallocates the image without changing the id.
```

Comment:
Shouldn’t this be triple slash?

Reply:
The type `ImageStore` isn't exported, so it doesn't need to be a doccomment. Could also be removed completely.